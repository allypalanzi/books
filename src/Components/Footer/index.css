.c-footer {
  border-top: 5px solid black;
  margin: 5% 0;
  text-align: center;
}