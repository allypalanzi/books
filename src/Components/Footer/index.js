import React from 'react';
import './index.css';

/*
* A header component renders the application header
*/

const Footer = props => {
  return (
    <footer className="c-footer">
      <p>Â©2018, Thanks Google for your Books API</p>
    </footer>
  );
}

export default Footer;
