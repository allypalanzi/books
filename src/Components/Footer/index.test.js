import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Footer from './';

describe('<Footer/ >', () => {
  const component = (<Footer />);
  const renderedComponent = renderer.create(component);

  it('renders correctly', () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
