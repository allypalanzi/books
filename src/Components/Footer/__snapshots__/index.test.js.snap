// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/ > renders correctly 1`] = `
<footer
  className="c-footer"
>
  <p>
    Â©2018, Thanks Google for your Books API
  </p>
</footer>
`;
